docs/gallery/README.md

```diff
@@ -0,0 +1,24 @@
+# Gallery
+
+Here is the collection of community-submitted examples of prompt's theme by using Starship.
+
+## How to Walk Through the Gallery
+
+In this section, you can:
+
+- see many themes
+- download the configuration file of Starship called `starship.toml` of each theme
+- see `starship.toml` of each theme on GitHub
+
+When you mouse over the image of each theme, two buttons appear: "Download the starship.toml" and "View the starship.toml on GitHub".
+
+If you are a newcomer to Starship and wanna use themes in the Gallery, please see the [Guide - Installation](http://localhost:8080/guide/#%F0%9F%9A%80-installation) and [Configuration](/config/) at first.
+
+## Contribution to the Gallery
+
+If you wanna add your theme to the Gallery, you can do this by creating a pull request to [the repository of Starship](https://github.com/starship/starship) on GitHub.
+Details are in our [Contributing Guide - Add the Theme of Your Prompt to the Gallery](https://github.com/starship/starship/blob/master/CONTRIBUTING.md#add-the-theme-of-your-prompt-to-the-Gallery).
+
+So let's walk through the Gallery!
+
+<Gallery/>
```

Comment:
Hi Mick(@mickimnet).

Thanks for your like👍

> Just one question remains: what about the Download instruction section? In my opinion, that would be helpful.

That's right. An instruction section is necessary.

I modified README to add the instruction section named How to Walk Through the Gallery.
This section explains how to use the Gallery.
And add another section _Contribution to the Gallery_ linked to `CONTRIBUTING.md`.

How about this?

Reply:
Hello Yosuke (@nukopy),

if I may ask, if it would be Ok for you, if we would provide you with amended versions of your texts in the `Gallery.md` and the `CONTRIBUTING.md` files?

Thank you very much and take care
Mick